src/processing/vFramer/src/vFramerLite.cpp

```diff
@@ -67,6 +68,12 @@ vDraw * createDrawer(std::string tag)
         return new skinDraw();
     if(tag == skinsampleDraw::drawtype)
         return new skinsampleDraw();
+    if(tag == isoDrawSkin::drawtype)
```

Comment:
Wouldn't it be more convenient to change these multiple `if` statements into a `switch case` statement?
It would improve the readability and it should be faster (not always true).

Reply:
you can't switch on a std::string